projects/demo/src/modules/app/app.routes.ts

```diff
@@ -1972,8 +1972,9 @@ export const ROUTES: Routes = [
 @NgModule({
     imports: [
         RouterModule.forRoot(ROUTES, {
-            initialNavigation: `enabledBlocking`,
             scrollPositionRestoration: `enabled`,
+            anchorScrolling: `enabled`,
+            scrollOffset: [0, 64],
```

Comment:
Should we drop this property ? 🤔 

https://github.com/taiga-family/taiga-ui/blob/3ef4f4892ff6a01333ca8a185bd8de283c675d62/projects/addon-doc/components/main/main.style.less#L6

Reply:
64px === 4rem